Examples/Examples-iOS/IGListKitExamples/Views/LabelCell.swift

```diff
@@ -26,7 +26,7 @@ final class LabelCell: UICollectionViewCell {
 
     static func textHeight(_ text: String, width: CGFloat) -> CGFloat {
         let constrainedSize = CGSize(width: width - insets.left - insets.right, height: CGFloat.greatestFiniteMagnitude)
-        let attributes = [ NSFontAttributeName: font ]
+        let attributes = [ NSAttributedStringKey.font: font ]
```

Comment:
Infer enum `.font`

Reply:
this won't work unless you explicitly add a type annotation:

`let attributes: [NSAttributedStringKey: Any] = [ .font: font]`

"swifty" style is to omit types, though, so probably leaves this as is.

Reply:
It would work if the attributes was inline with the actual initialiser which given the length of text probably wouldn't be bad

Not fussed either way

Reply:
Ah yes, 👍👍👍